projects/github.com/jaraco/keyring/package.yml

```diff
@@ -0,0 +1,25 @@
+distributable:
+  url: git+https://github.com/jaraco/keyring.git
+  ref: ${{version.tag}}
+
+versions:
+  github: jaraco/keyring
+
+dependencies:
+  pkgx.sh: ^1
+  github.com/python-cffi/cffi: ^1.16
+  github.com/eliben/pycparser: ^2.21
+  cryptography.io: ^42
+
+build:
+  dependencies:
+    python.org: '>=3.11'
```

Comment:
This is more acceptable because it is a build dependency. but at the same time it will result in build failures sporadically and it is better to constrain python correctly